Mycale development

mcradcliffe2490 opened the pull request:
Gives me the base portfolio website after the tutorial. From here I can personalize the website the way I want.

mcradcliffe2490 committed:
Update the README.md file and show that I have successfully linked the repository

mcradcliffe2490 committed:
Gives me base website after the tutorial to start making personalized changes from

mcradcliffe2490 committed:
Gives the website a background and a banner across the top of the site

mcradcliffe2490 committed:
Adds more structure to website and another picture

mcradcliffe2490 committed:
Adds some social buttons to the banner of the webpage that will be updated with functionality later

Snehalb26 commented on portfolio/src/main/webapp/index.html:
```diff
@@ -8,11 +8,15 @@
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1>Mycale's portfolio</h1>
       <p>This is my portfolio.</p>
```
This can be removed

Snehalb26 commented on portfolio/src/main/webapp/script.js:
```diff
@@ -13,16 +13,30 @@
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomQuote() {
+  const quotes =
+      ["If you're the CEO of a company, and you're dumb enough to leave your log-in info on a Post-It note on your desk, while the people that you ripped off are physically in your office, it's not a hack. It's barely social engineering. It’s more like natural selection." ,
```
It is in general good practice to move text on next line if character count exceeds 100. This helps in code readability.

Snehalb26 commented on portfolio/src/main/webapp/script.js:
```diff
@@ -13,16 +13,30 @@
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const QuoteContainer = document.getElementById('quote-container');
```
rename this to 'quoteContainer' to follow naming conventions.

Snehalb26 commented on portfolio/src/main/webapp/index.html:
```diff
@@ -7,8 +7,10 @@
+    <div id="banner">
+        <h1><strong> Mycale's Website</strong></h1> <a href="/images/sun.gif"><img src = "/images/sun.gif"/></a>
+    </div>
+    <div id="para1">
```
Can we use better name that para1 for the div id.

mcradcliffe2490 committed:
Readjust some elements on the Webpage and adds a short About Me Section

mcradcliffe2490 committed:
Deployed website and cleaned up code to match style guides

mcradcliffe2490 committed:
Addresses changes brought up in the comments such as:
- long lines of code in script.js
- name and other miscellaneous changes in index.html
